charts/pega/values.yaml

```diff
@@ -49,7 +49,10 @@ global:
       url: "YOUR_DOCKER_REGISTRY"
       username: "YOUR_DOCKER_REGISTRY_USERNAME"
       password: "YOUR_DOCKER_REGISTRY_PASSWORD"
-      # Docker image information for the Pega docker image, containing the application server.
+    # Optional config path to a secrets file if not using registry creds.
+    config:
+      path: ""
```

Comment:
Can this param just be omitted or is the empty string significant?

Reply:
If we leave the default empty string, can we add some comments about format and expected value of the path?  Can any path be specified or does the location need to be based on some existing root path?

Reply:
On the empty string: seen similar things done throughout this file and so copied that standard here for better/worse.

The "external-secrets" is an external project that can be found [HERE](https://github.com/godaddy/kubernetes-external-secrets) that we wrap internally as an SCE (ping me for name or just search "external" in bitbucket)

Reply:
I can put a comment to the relevant github project here and note how it's necessary for use if you're using this feature.

Reply:
@pega-embate just updated the documentation here to be as specific as possible. As this does require the use of an `kubernetes-externel-secrets`, which we use internally, I can see this _maybe_ being a potentially not desired change. If that's the case we could most likely modify our internal `SCE` to do this very thing if necessary instead of making the change here. I just figured this could be beneficial to the community.

@dcasavant any thoughts one way or another?